locale/defaultMessages.json

```diff
@@ -9262,6 +9265,9 @@
   "w7jT4W": {
     "string": "No channels selected"
   },
+  "w7tf2z": {
+    "string": "Published"
```

Comment:
suggestion: maybe we should use `yes/no` label values for this? When you look at the condition as a filter, then it looks incorrect:

![CleanShot 2025-02-12 at 16 43 49@2x](https://github.com/user-attachments/assets/2a56cb4e-7eec-455b-8f74-db15330f2c3f)

`Published is Published` 😅

Reply:
or maybe we can change the field name to `Visibility`, so that it becomes:

`Visibility is Published`
`Visibility is Hidden`